cli/test/clojure_lsp/main_test.clj

```diff
@@ -5,7 +5,7 @@
    [clojure.java.io :as io]
    [clojure.test :refer [deftest is testing]]))
 
-(h/reset-db-after-test)
+(h/reset-components-before-test)
```

Comment:
This has always been incorrectly named—the reset happened before each test, not after.